libqtile/scripts/main.py

```diff
@@ -0,0 +1,35 @@
+import argparse
+import os
+
+from libqtile.scripts import cmd_obj, run_cmd, shell, start, top
+
+try:
+    import pkg_resources
+    VERSION = pkg_resources.require("qtile")[0].version
+except (pkg_resources.DistributionNotFound, ImportError):
+    VERSION = 'dev'
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        prog='qtile',
+        description='A full-featured, pure-Python tiling window manager.',
+    )
+    parser.add_argument(
+        '--version',
+        action='version',
+        version=VERSION,
+    )
+
+    subparsers = parser.add_subparsers()
+    start.add_subcommand(subparsers)
+    shell.add_subcommand(subparsers)
+    top.add_subcommand(subparsers)
+    run_cmd.add_subcommand(subparsers)
+    cmd_obj.add_subcommand(subparsers)
+    options = parser.parse_args()
+    if not hasattr(options, "func"):
+        parser.print_help()
+        print("running qtile has moved to subcommand 'qtile start'")
+        os.exit(1)
```

Comment:
`AttributeError: module 'os' has no attribute 'exit'`. Did you mean sys.exit?

Reply:
I sure did; now why didn't our CI catch this?